### PR TITLE
[Core] Add FindIntersectedGeometricalObjectsProcess to python

### PR DESCRIPTION
**Pull request by miguelmaso**
**📝 Description**
I would like to call this process from python

**🆕 Changelog**
- Add FindIntersectedGeometricalObjectsProcess to python

### CODE REVIEW

**Review by loumalouomega — DISMISSED** [review]
okay

File: kratos/python/add_processes_to_python.cpp [hunk 1]
```diff
@@ -186,6 +187,14 @@ void  AddProcessesToPython(pybind11::module& m)
     .def("GetNeighbourIds",&FindGlobalNodalElementalNeighboursProcess::GetNeighbourIds)
     ;
 
+    py::class_<FindIntersectedGeometricalObjectsProcess, FindIntersectedGeometricalObjectsProcess::Pointer, Process>
+        (m, "FindIntersectedGeometricalObjectsProcess")
+    .def(py::init<Model&, Parameters>())
+    .def(py::init<ModelPart&, ModelPart&>())
+    .def(py::init<ModelPart&, ModelPart&, const Flags>())
```
Comment (rubenzorrilla) [inline kratos/python/add_processes_to_python.cpp]: Working towards the integration of factories I wouldn't add these two if they are not strictly needed.
```suggestion
```

File: kratos/python/add_processes_to_python.cpp [hunk 2]
```diff
@@ -186,6 +187,14 @@ void  AddProcessesToPython(pybind11::module& m)
...
+    .def("FindIntersections", &FindIntersectedGeometricalObjectsProcess::FindIntersections)
```
Comment (rubenzorrilla) [inline kratos/python/add_processes_to_python.cpp]:
```suggestion
```
I think this shouldn't be added (the `Execute` should be sufficient).

File: kratos/python/add_processes_to_python.cpp (hunk 1 above)
Comment (miguelmaso) [reply kratos/python/add_processes_to_python.cpp]: removed

File: kratos/python/add_processes_to_python.cpp (hunk 2 above)
Comment (miguelmaso) [reply kratos/python/add_processes_to_python.cpp]: removed

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**